src/ir_passes/mono/monomorphize.rs

```diff
@@ -40,14 +20,11 @@ pub struct Monomorphize<'a> {
 
     /// Instances that have already been processed. Tracks the name of the generated component
     pub processed: HashMap<CompKey, Base<ir::Component>>,
-    /// Instances that need to be generated
-    pub queue: LinkedHashMap<CompKey, (Base<ir::Component>, MonoSig)>,
 
     /// Mapping from old ports to new ports, for resolving Foreigns
-    pub port_map: HashMap<PortKey, Base<ir::Port>>,
-    /// Mapping from old events to new events, for resolving Foreigns
-    pub event_map: HashMap<EventKey, Base<ir::Event>>,
+    inst_info: HashMap<CompKey, InstanceInfo>,
```

Comment:
comment?

Reply:
There is a comment about `inst_info` already. Do you mean something else?